src/components/framework/head.js

```diff
@@ -3,7 +3,25 @@ import { connect } from "react-redux";
 import { Helmet } from "react-helmet";
 import { hasExtension, getExtension } from "../../util/extensions";
 
-const Head = ({metadata}) => {
+import i18n from "i18next";
+import { initReactI18next } from "react-i18next";
+
+const Head = ({metadata, general}) => {
+  let lang = general.language;
+
+  i18n
+  .use(initReactI18next)
+  .init({
+    resources: require("i18next-resource-store-loader!../../locales/index.js"),
```

Comment:
Review app error here:

```
ERROR in ./src/components/framework/head.js

Module not found: Error: Can't resolve '../../locales/index.js' in '/home/travis/build/nextstrain/auspice/src/components/framework'
...
```

Reply:
Oh, you know, I should include the language files in the PR 😅